### PR TITLE
RHDEVDOCS-2831: Add module for editing deployments

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/RHDEVDOCS-2831

Link to docs preview:
- https://61709--docspreview.netlify.app/openshift-enterprise/latest/applications/deployments/deployment-strategies.html#odc-editing-deployments_rolling-strategy
- https://61709--docspreview.netlify.app/openshift-enterprise/latest/applications/deployments/deployment-strategies.html#odc-editing-deployments_recreate-strategy (content reuse)
- https://61709--docspreview.netlify.app/openshift-enterprise/latest/applications/deployments/deployment-strategies.html#odc-editing-deployments_custom-strategy (content reuse)
- https://61709--docspreview.netlify.app/openshift-enterprise/latest/applications/deployments/deployment-strategies.html#odc-starting-rolling-deployment_rolling-strategy (editing only)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by christoph-jerolimov — APPROVED** [review]
Lgtm!

**Comment by sanketpathak** [comment]
QE approved

File: modules/odc-editing-deployments.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* You have created an application.
+
+.Procedure
+
+. Navigate to the *Topology* view. Click on your application to see the *Details* panel.
+. In the *Actions* drop-down menu, select *Edit Deployment* to view the *Edit Deployment* page.
+. You can edit the following *Advanced options* for your deployment:
+.. Optional: You can pause rollouts by clicking *Pause rollouts*, and then selecting the *Pause rollouts for this deployment* checkbox.
++
+Pausing rollouts allows you to make changes to your application without triggering a rollout. You can resume rollouts at any time.
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply modules/odc-editing-deployments.adoc:23]: I always forget this one, thank you!

**Comment by bburt-rh** [comment]
One small nitpicky suggested edit, but otherwise lgtm

**Review by bburt-rh — APPROVED** [review]

**Pull request merged**